### DISCUSSION TITLE
Feedback on the design document

### DISCUSSION THREAD

**Discussion by benbovy** [xarray-contrib/xdggs#22] [General]
I'm opening this discussion to collect feedback on the design document about an Xarray extension for [Discrete Global Grid Systems (DGGSs)](https://en.wikipedia.org/wiki/Discrete_global_grid) (this repository).

The document can be found here: https://github.com/benbovy/xdggs/blob/main/design_doc.md

**Comment by danlooo** [comment]
I started PR #24 to update the design document
